modules/twinklearv.js

```diff
@@ -611,6 +641,54 @@ Twinkle.arv.callback.evaluate = function(e) {
 				aivPage.append();
 			});
 			break;
+		// Report 3RR
+		case 'ewip':
+			if (comment === '') {
+				alert(wgULS('您必须指定理由', '您必須指定理由'));
+				return;
+			}
+			header = '\n=== {{vandal|' + (/=/.test(uid) ? '1=' : '') + uid + '}} ===\n';
+
+			var pages = $.map($('input:text[name=page]', form), function (o) {
+				return $(o).val() || null;
+			});
+			if (pages.length >= 1) {
+				pages.forEach(function(v) {
+					reason += '* {{pagelinks|' + v + '}}\n';
+				});
+			}
+			reason += '* ' + comment + '\n';
+			reason = reason.trim();
+			reason = Morebits.string.appendPunctuation(reason);
+			reason += wgULS('\n* 提报人：~~~~\n* 处理：', '\n* 提報人：~~~~\n* 處理：');
```

Comment:
與站內模板保持一致，不提供繁簡轉換，否則可能會影響小工具/防濫用過濾器
```suggestion
			reason += '\n* 提報人：~~~~\n* 处理：';
```